crds/objectstorage.k8s.io_bucketrequests.yaml

```diff
@@ -42,48 +42,16 @@ spec:
                 type: string
               protocol:
                 properties:
-                  azureBlob:
-                    properties:
-                      containerName:
-                        type: string
-                      storageAccount:
-                        type: string
-                    type: object
-                  gcs:
-                    properties:
-                      bucketName:
-                        type: string
-                      privateKeyName:
-                        type: string
-                      projectID:
-                        type: string
-                      serviceAccount:
-                        type: string
-                    type: object
-                  protocolName:
+                  name:
                     enum:
                     - s3
                     - azure
                     - gcs
                     type: string
-                  s3:
-                    properties:
-                      bucketName:
-                        type: string
-                      endpoint:
-                        type: string
-                      region:
-                        type: string
-                      signatureVersion:
-                        enum:
-                        - s3v2
-                        - s3v4
-                        type: string
-                      version:
-                        type: string
-                    type: object
+                  version:
```

Comment:
Does it matter that the API defines this as "azureBlob" on L48?

Reply:
@rrati This is how you named, not sure of the context. 'protocol: azure' and the struct that container details protocol struct named as 'azureBlob'.